provision/infinispan/README.adoc

```diff
@@ -2,55 +2,26 @@
 
 Deploy an Infinispan cluster (with or without cross-site) and configures the necessary caches for Keycloak.
 
-== Required variables
+== Taskfile
 
-* `CLUSTER_1`: The ROSA cluster name of site 1.
-* `CLUSTER_2`: The ROSA cluster name of site 2.
-* `NS_1`: The namespace to install Infinispan in site 1.
-* `NS_2`: The namespace to install Infinispan in site 2.
+For single cluster mode:
 
-== Optional variables:
-* `KUBECONFIG_1`: Path to `KUBECONFIG` to use for CLUSTER_1.
-* `KUBECONFIG_2`: Path to `KUBECONFIG` to use for CLUSTER_2.
-
-If file does not exist, rosa_oc_login will be invoked and the configuration stored in this path.
-
-== Infinispan cluster customization:
-
-* `XSITE_MODE`: The cross-site strategy, default to SYNC.
-* `ISPN_REPLICAS`: The number of Infinispan pods.
-
-== Operation Modes
-
-* CLUSTER_1 == CLUSTER_2 and NS_1 == NS_2 -> Single Infinispan cluster without cross-site.
-
-Example:
-[source, bash]
-----
-CLUSTER_1="gh-pruivo" CLUSTER_2="gh-pruivo" NS_1="ispn-server" NS_2="ispn-server" ./create_ispn_clusters.sh
-----
-
-* CLUSTER_1 == CLUSTER_2 and NS_1 != NS_2 -> Infinispan clusters with cross-site enabled in a single OCP cluster.
-Each namespace gets an Infinispan cluster, and they are linked together with cross-site.
-
-Example:
 [source, bash]
 ----
-CLUSTER_1="gh-pruivo" CLUSTER_2="gh-pruivo" NS_1="server-site-1" NS_2="server-site-2" ./create_ispn_clusters.sh
+task single-cluster ROSA_CLUSTER_NAME=gh-keycloak OC_NAMESPACE=infinispan
 ----
 
-* CLUSTER_1 != CLUSTER_2 -> Infinispan clusters with cross-site enabled in 2 different OCP clusters.
-The namespaces must be set and they can be the same or different namespaces.
+For cross-site between different namespaces:
 
-Example:
 [source, bash]
 ----
-CLUSTER_1="gh-pruivo" CLUSTER_2="gh-keycloak" NS_1="ispn-server" NS_2="ispn-server" ./create_ispn_clusters.sh
+task crossdc-single ROSA_CLUSTER_NAME=gh-keycloak OC_NAMESPACE_1=ispn-1 OC_NAMESPACE_2=ispn-2
```

Comment:
can we re-use the existing infinispan namespaces to run this command ?

Reply:
I've been using the same namespace for keycloak and infinispan without any issues.